src/i18n/de-DE.i18n.json

```diff
@@ -4,7 +4,7 @@
     "copy": "&Kopieren",
     "paste": "&Einfügen",
     "selectAll": "Alle auswählen",
-    "undo": "Wider&rufen",
+    "undo": "Wieder&rufen",
```

Comment:
I don't know German so much, but aren't you introducing a typo here?

Reply:
Sure, sorry my mistake, didn't know what io was thinking when i saw it. You can delete my Pull request

Reply:
No problem! 👍